snaps/reference/snaps-api.md

```diff
@@ -10,9 +10,52 @@ import TabItem from '@theme/TabItem';
 # Snaps API
 
 Snaps can communicate with and modify the functionality of MetaMask using the [Snaps API](../learn/about-snaps/apis.md#snaps-api).
-To call each method, you must first [request permission](../how-to/request-permissions.md) in the Snap
+To call some of these methods, you must first [request permission](../how-to/request-permissions.md) in the Snap
```

Comment:
To clarify, the three new `Interface` methods do NOT require requesting permission in `initialPermissions`, but the rest of the methods do?

Reply:
`snap_createInterface`, `snap_updateInterface`, `snap_getInterfaceState` and the `onUserInput` handler are all permitted meaning there's no permission needed to use those.